job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/ScheduledExecutor.java

```diff
@@ -68,7 +70,7 @@ public static void runAvailableJobs(final String origin)
             } else {
                 DatabasePoller.pollDatabaseForJobsToRun();
             }
-        } catch (final Exception t) {   // Catch Exceptions and Errors to prevent scheduler stoppage.
+        } catch (final Throwable t) {   // Catch Exceptions and Errors to prevent scheduler stoppage.
```

Comment:
I notice that this was changed in [#252](https://github.com/JobService/job-service/pull/252/files#diff-b919fd6160681b0f5ef4415ea692e16fed5de0a59093e049e0efb7bddae25ea8L39).  I'm not sure why this was changed so I'm just putting it back.